openpype/hosts/blender/api/lib.py

```diff
@@ -169,6 +169,8 @@ def update_scene_containers() -> List[OpenpypeContainer]:
         if (
             entity in datablocks_to_skip
             or user_map.get(entity) <= datablocks_to_skip
+            if user_map.get(entity)
+            else False
```

Comment:
I think this should be an `and` instead of an `if` `else` statement inside of another `if`